src/main/features/claimant-response/views/check-and-send.njk

```diff
@@ -66,15 +66,15 @@
 
     {{ tableEnd() }}
 
-    {{ tableStart('howYouWnatToProceed', 'How you wish to proceed') }}
     {% if draft.formaliseRepaymentPlan %}
-      {{ row(
-        label = 'How do you want to formalise the repayment plan?',
-        bold = true,
-        value = draft.formaliseRepaymentPlan.option | renderFormaliseRepaymentPlanOption,
-        changeLink = ClaimantResponsePaths.chooseHowToProceedPage.evaluateUri({ externalId: claim.externalId }), bottomBorder = true
-      ) }}
-    {% endif %}
+      {{ tableStart('howYouWantToProceed', 'How you wish to proceed') }}
```

Comment:
Please move `{{ tableEnd() }}` inside if statement as well. At the moment `tableStart` is conditional and `tableEnd` is rendered always